src/ert/_c_wrappers/enkf/data/enkf_node.py

```diff
@@ -45,20 +45,13 @@ class EnkfNode(BaseCClass):
         "ert_impl_type_enum enkf_node_get_impl_type(enkf_node)"
     )
 
-    def __init__(self, config_node: "EnkfConfigNode", private: bool = False):
-        self._private = private
-        if private:
-            c_pointer = self._alloc_private(config_node)
-        else:
-            c_pointer = self._alloc(config_node)
+    def __init__(self, config_node: "EnkfConfigNode"):
+        c_pointer = self._alloc(config_node)
```

Comment:
I could not find any single instance of `EnkfNode` specifying explicitly a value for `private`. 
Could you please explain why was that used? Any idea why we don't have any single test to clean up?

Reply:
Correct, there is nothing that uses this feature (and if used, it would crash), so I removed it.